Corona-Warn-App/src/main/res/layout/contact_diary_edit_list_item.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/item_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="4dp"
+    style="@style/cardGrey"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+        style="@style/subtitle"
+        android:id="@+id/name"
+        android:layout_width="0dp"
```

Comment:
Same here.
You are sometimes using layout_width="0dp" and sometimes layout_width=@dimen/match_constraint" for Textview or other Views. Please decide for one way.